src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -227,6 +241,9 @@ export const NavBar: React.FC<NavBarPropsType> = ({
 
               {shouldShowSchedule ? (
                 <div
+                  role="button"
+                  tabIndex={0}
+                  onKeyPress={(e) => handleKeyPress(e, toggleEventSchedule)}
```

Comment:
We shouldn’t need to explicitly handle key presses here if we use the correct base types that naturally support these navigations. Eg. Button